Userland/Libraries/LibPDF/CommonNames.h

```diff
@@ -37,6 +37,7 @@
     X(CIDFontType2)               \
     X(CIDSystemInfo)              \
     X(CIDToGIDMap)                \
+    X(CharProcs)                  \
```

Comment:
Nit-pick: h comes before i. :wink:

Reply:
Aha, but upper-case I comes before lower-case h!

(This list is sorted by ascii value without case folding.)

Reply:
Then what's happening with the Cal... ones above? :thonk:

Reply:
Good catch, I did insert _those_ in the wrong position.

Reply:
🆘🚨➡ #21974